src/helpers/SocialReferrer.js

```diff
@@ -4,18 +4,25 @@ var SocialReferrer = {
    *
    * @returns {String};
    */
-  getReferrer: function() {
+  getReferrer: function () {
     return window.document.referrer || '';
   },
  /**
-  * Determines if the referrer is social media
+  * Determines if the referrer is social media.
+  * If true, we want to return the type of social media
   *
   * @returns {Boolean};
   */
-  isSocialReferrer: function() {
-    var socialReferrer = this.getReferrer().match(/\b(?:facebook|instagram|pinterest|reddit|twitter|tumblr|t\.co)\b/i);
+  isSocialReferrer: function () {
```

Comment:
We should change the method signature to something other than `isSocialReferrer` now that it returns a string value (should also update the comment too)